tools/bumper/cnaoRepoCommands_test.go

```diff
@@ -0,0 +1,277 @@
+package main
```

Comment:
move this file to cnaorepocommaneds_test.go

Reply:
changed to cnao_repo_commands_test.go
Done